pkg/cluster/dashboards_list.go

```diff
@@ -0,0 +1,54 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/netobserv/network-observability-operator/controllers/constants"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+)
+
+var orderedDashboards = []string{
+	constants.KubernetesNetworkDashboard,
+	constants.FlowDashboardCMName,
+	constants.HealthDashboardCMName,
+	constants.IngressDashboardCMName,
+	constants.NetStatsDashboardCMName,
+	constants.OVNDashboardCMName,
+}
+
+type Dashboards struct {
+	availableCMNames map[string]interface{}
+}
+
+func NewDashboards() Dashboards {
+	return Dashboards{
+		availableCMNames: map[string]interface{}{
+			constants.KubernetesNetworkDashboard: nil,
+		},
+	}
+}
+
+func (d *Dashboards) CheckClusterDashboards(ctx context.Context, clusterInfo *Info) {
+	if ok, err := clusterInfo.OpenShiftVersionIsAtLeast("4.15.0"); err != nil {
+		// Log error but do not fail: it's likely a bug in code, if the openshift version cannot be found
+		log.FromContext(ctx).Error(err, "Could not get available dashboards for this cluster version. Is it OpenShift?")
```

Comment:
Nit: log info including `err.Message` instead ?

Reply:
if fact this function is only called for openshift (on the caller's side, it's inside a `if c.HasOCPSecurity()` block, which is our way to detect openshift). But I agree it could be made more explicit, let me see that...